src/geolocation.re

```diff
@@ -0,0 +1,104 @@
+type rnConfig;
```

Comment:
do these last two files (re, rei) need to be capitalized? Geolocation.re?

Reply:
They don't need to, and probably should stay lower cased in convention with everything else in the src folder.

Reply:
What is rnConfig?

Reply:
it's an opaque type, look here: https://github.com/reasonml-community/bs-react-native/pull/162/files#diff-54cf7aeba2428bd523fa4fd98951effeR30

I guess the name is a bit confusing, do you have any suggestion?

Reply:
Yeah I get that it's an opaque type, just wondering what it's used for. It's just the name that's confusing me. Could you link me to the react native docs for this so I can see what it's intended for 😄